Backendless/Persistence/DataQueryBuilder.cs

```diff
@@ -17,6 +17,8 @@ private DataQueryBuilder()
       properties = new List<String>();
       pagedQueryBuilder = new PagedQueryBuilder<DataQueryBuilder>( this );
       queryOptionsBuilder = new QueryOptionsBuilder<DataQueryBuilder>( this );
+      groupBy = new List<String>();
+      havingClause = "";

```

Comment:
if someone uses `SetHavingClause( null )`, it will overwrite the value and break the API